variamosple/VariaMosPLE
# Adding DB user profile to the session storage

# Description

Including the database gathered user profile to the session storage.

# Test
1. Open the Developer Tools panel.
2. Run `JSON.parse(sessionStorage.databaseUserProfile)`
3. It should show the database stored User Profile
